modules/go-run-argo-cd-instance-on-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * gitops/configuring_argo_cd_to_recursively_sync_a_git_repository_with_your_application/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+[id="run-argo-cd-instance-on-cluster_{context}"]
+
+= Running the Argo CD instance at the cluster-level
+
+The default Argo CD instance installed by the {gitops-title} as well as accompanying controllers can now run on the infrastructure nodes of the cluster by setting a simple configuration toggle.
```

Comment:
by the {gitops-title} Operator as well as the accompanying

Reply:
yes @Preeticp your suggestion looks good.